### PR TITLE
add sort by area (largest to smallest)

### PR DESCRIPTION
**Pull request by Tonny2442**
wanted to add sort by area, sort method may not be optimized but it works!

### CODE REVIEW

File: web/_js/view.js [hunk 1]
```diff
@@ -449,6 +449,21 @@ function initView(){
...
+					let areaB = calcPolygonArea(b.path);
+					if (areaA > areaB) {
+						return -1;
+					}
+					if (areaA < areaB) {
+						return 1;
+					}
+						// areaA must be equal to areaB
+					return 0;
+				}
```
Comment (TrevorSayre) [inline web/_js/view.js:465]: should be able to `return areaB - areaA`

Comment (Tonny2442) [reply web/_js/view.js:456]: 👍thanks

**Review by ProstoSanja — COMMENTED** [review]
I'm happy to merge this, if you put this PR and #822 together in one.
NIce and easy solution for optimisation there!

**Comment by Tonny2442** [comment]
> I'm happy to merge this, if you put this PR and #822 together in one. NIce and easy solution for optimisation there!

Thank You! moved #822 here.

**Review by imskyyc — APPROVED** [review]

**Review by ProstoSanja — APPROVED** [review]

**Review by Codixer — APPROVED** [review]

**Comment by ProstoSanja** [comment]
Might want to create a separate PR to sort by area the other way round, small to large.

**Pull request merged**